tommath_private.h

```diff
@@ -16,7 +16,7 @@
  * as a shared object. By default, symbols are visible.
  * On Win32 a .def file must be used to specify the exported symbols.
  */
-#if defined(__GNUC__) && __GNUC__ >= 4 && !defined(_WIN32)
+#if defined(__GNUC__) && __GNUC__ >= 4 && !defined(_WIN32) && !defined(__CYGWIN__)
```

Comment:
is this patch something that should also be in 1.2? if yes, can you please make it a separate patch so I can cherry-pick it for an eventual 1.2.1

Reply:
That wouldn't work anyway, because on 1.2.0 the #ifdef is different:
    #if defined (MP_PRIVATE_SYMBOLS) && defined(__GNUC__) && __GNUC__ >= 4
    #   define MP_PRIVATE __attribute__ ((visibility ("hidden")))
    #else
    #   define MP_PRIVATE
    #endif

Reply:
OK, so in 1.2.0 it should be `#if defined (MP_PRIVATE_SYMBOLS) && defined(__GNUC__) && __GNUC__ >= 4 && !defined(_WIN32) && !defined(__CYGWIN__)` right?